Automattic/wp-calypso
# Nav Unification: Remove old sidebar (part 5)

This PR is part of a bigger cleanup task which seeks to remove no longer used code after the Nav Unification launch (see  https://github.com/Automattic/wp-calypso/issues/47051).

#### Changes proposed in this Pull Request

Removes the `_nav-unification.scss` file which contained styles overrides for Nav Unification and move them to the appropriate files so they can be maintained with a _Nav Unification-first_ approach.

<details>
<summary><strong>Context</strong></summary>

Last year, we introduced new components for the navigation sidebar as part of the Nav Unification project. However, we've been falling back to the old sidebar components in these scenarios:
- As an escape hatch using [the `?disable-nav-unification` query param](https://github.com/Automattic/wp-calypso/blob/c7e06d5e2fe1ca94643d16855209704c39452f30/client/state/selectors/is-nav-unification-enabled.js#L9-L12).
- For self-hosted Jetpack sites [not updated to Jetpack 9.8](https://github.com/Automattic/wp-calypso/blob/c7e06d5e2fe1ca94643d16855209704c39452f30/client/state/selectors/is-nav-unification-enabled.js#L14-L19).
- [In Jetpack Cloud](https://github.com/Automattic/wp-calypso/blob/c7e06d5e2fe1ca94643d16855209704c39452f30/client/my-sites/navigation/index.jsx#L24-L25).

We no longer need to support the first two scenarios, since the escape hatch was meant to be temporary and Calypso does not need to support lower Jetpack versions than 9.8 anymore.

As for the third scenario, Jetpack Cloud only needs a small subset of the old components.

That gives us a good opportunity to remove a lot of unused code which would simplify the overall Calypso codebase.
</details>

#### Testing instructions

- Use the live links below for both Calypso and Jetpack Cloud.
- Make sure there are no regressions in the sidebar and/or the masterbar.
